openslides/users/static/js/users/site.js

```diff
@@ -912,6 +907,9 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
                     // Success: User logged in.
                     operator.setUser(response.data.user_id);
                     $scope.closeThisDialog();
+                    setTimeout(function(){
```

Comment:
I tested it but it does not work nicely because we have to wait until the dialog is closed and then reload  the page. Otherwise you has unattractiv effects if you reload if the dialog is still open.
So: won't fix!